src/Reducer/Video-Reducer.js

```diff
@@ -0,0 +1,102 @@
+import { searchMyNotes } from "../ReusableFunctions/funcs";
+
+const ADD_TO_LIKES = "ADD_TO_LIKES";
+const REMOVE_FROM_LIKES = "REMOVE_FROM_LIKES";
+const ADD_NOTE = "ADD_NOTE";
+const ADD_NEW_PLAYLIST = "ADD_NEW_PLAYLIST";
+const ADD_TO_PLAYLIST = "ADD_TO_PLAYLIST";
+const REMOVE_FROM_PLAYLIST = "REMOVE_FROM_PLAYLIST";
+const DELETE_PLAYLIST = "DELETE_PLAYLIST";
+const SEARCH_FILTER = "SEARCH_FILTER";
+
+
+export function dispatchFunc(state , { type, payload }) {
+    switch (type) {
+        case ADD_TO_LIKES:
+            return { ...state, likedVideo: [...state.likedVideo, payload] }
+        
+        case REMOVE_FROM_LIKES:
+            return { ...state, likedVideo: state.likedVideo.filter(item => item.id !== payload.id) }
+        
+        case ADD_NOTE:
+            if (searchMyNotes(state,payload)){
+                return {
+                    ...state, myNotes : state.myNotes.map(one => {
+                        if (one.id === payload.id)
+                            return { ...one, notes: [...one.notes, payload.input] }
+                        return one
+                    })
+                }
+            } else {
+                return {...state, myNotes : [...state.myNotes, {id : payload.id, notes : [payload.input]}]}
+            }
+            
+        case ADD_NEW_PLAYLIST:
```

Comment:
Can you send `payload` as `{ playlistName: 'XYZ' }` so that here you can assign as `{ name: payload.playlistName } which is more meaningful and understandable